x86/Uci.asm

```diff
@@ -1145,7 +1157,9 @@ UciBench:
            call  CmpString
            test  eax, eax
             jnz  .parse_hash
-            jmp  .parse_done
+           call  CmpString
+           test  eax, eax
+	        jmp .parse_done
```

Comment:
The call and test do nothing here. Calling CmpString is also bad unless you know that rcx holds the address of a cstring.